src/include/gurt/common.h

```diff
@@ -586,6 +586,9 @@ int
 d_getenv_bool(const char *name, bool *bool_val);
 int
 d_getenv_char(const char *name, char *char_val);
+/* DAOS-14981 XXX d_getenv_int() is deprecated, please use d_getenv_uint() */
+int
+d_getenv_int(const char *name, unsigned int *uint_val);
```

Comment:
I'm not at my desk so can't try it but this should work.
```suggestion
int
d_getenv_int(const char *name, unsigned int *uint_val) __attribute__ ((deprecated(use d_getenv_uint));
```

Reply:
- [ ] Use compiler intrinsic macro.

Reply:
> * [x]  Use compiler intrinsic macro.

Fixed with commit e95531c0de297d42cdb888cb34db66305a34003c